sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeServiceClient.cs

```diff
@@ -548,13 +548,13 @@ public virtual DataLakeFileSystemClient GetFileSystemClient(string fileSystemNam
         /// <param name="publicAccessType">
         /// Optionally specifies whether data in the file system may be accessed
         /// publicly and the level of access. <see cref="PublicAccessType.FileSystem"/>
-        /// specifies full public read access for file system and blob data.
-        /// Clients can enumerate blobs within the file system via anonymous
+        /// specifies full public read access for file system and path data.
+        /// Clients can enumerate paths within the file system via anonymous
         /// request, but cannot enumerate file systems within the storage
         /// account.  <see cref="PublicAccessType.Path"/> specifies public
```

Comment:
Are we okay with stating that these are file systems within the account even though it's just a blob account being accessed a certain way? I don't have an alternative, just calling it out to see if there's a better option.

Reply:
We refer the container as "file system" and the blob as "path/file/directory" through the Data Lake SDK.